src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,45 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run floodLight in the debug mode we have to add this line of arguments
+```
+"-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y”
+``` 
+when running the app.
+Particularly in floodLight we can not do this straight to the Dockerfile since it leads us to entrypoint.sh script.
+We have to add these arguments to the
+```
+projectroot/docker/floodlight-modules/entrypoint.sh
+```
+file;
+
+
+in this file find `exec java` line under the ```"$1" = 'floodlight’```  condition.
+In the end it should look something like this:
+```
+exec java "-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y" -XX:+PrintFlagsFinal……… the rest arguments….;
+```
+then we have to expose debugging port in docker-compose.tmpl file.
+go to ```projectroot/confd/templates/docker-compose/docker-compose.tmpl```
+find floodlight_1 and add desirable port mapping:
```

Comment:
It is a list of actions. You need a comma between them: go, find, and add. (the comma before and as well). `port` is countable and requires an article: a desirable port.

Reply:
thanks, fixed

Reply:
`go, find, and add` notice there is a comma before `and` as well. This is called an Oxford comma

Reply:
fixed thank you.